modules/combined/test/tests/sliding_block/in_and_out/constraint/frictionless_kinematic_out.cmp

```diff
@@ -0,0 +1,47 @@
+
+#  *****************************************************************
+#             EXODIFF	(Version: 2.83) Modified: 2015-08-20
+#             Authors:  Richard Drake, rrdrake@sandia.gov           
+#                       Greg Sjaardema, gdsjaar@sandia.gov          
+#             Run on    2018/04/23   15:43:52 MDT
+#  *****************************************************************
+
+#  FILE 1: /Users/kongf/projects/moose/modules/combined/test/tests/sliding_block/in_and_out/constraint/frictionless_kinematic_out.e
+#   Title: frictionless_kinematic_out.e
+#          Dim = 2, Blocks = 2, Nodes = 224, Elements = 146, Nodesets = 4, Sidesets = 4
+#          Vars: Global = 3, Nodal = 9, Element = 0, Nodeset = 0, Sideset = 0, Times = 16
+
+
+# ==============================================================
+#  NOTE: All node and element ids are reported as global ids.
+
+# NOTES:  - The min/max values are reporting the min/max in absolute value.
+#         - Time values (t) are 1-offset time step numbers.
+#         - Element block numbers are the block ids.
+#         - Node(n) and element(e) numbers are 1-offset.
+
+COORDINATES absolute 1.e-6    # min separation not calculated
+
+TIME STEPS relative 1.e-6 floor 0.0     # min:               0 @ t1 max:              15 @ t16
+
+GLOBAL VARIABLES absolute 1.e-7 floor 0.0
+	contact_pressure  # min:               0 @ t1	max:       10745.055 @ t3
```

Comment:
This is fancy, but can we just use `outputs = console` in that postprocessor to not output it into exodus file?

Reply:
We talked about this, but we assumed that the nonlinear iterations were being output for a reason and we didn't want to change the intention of the original tests. Instead we just wanted to ignore data that doesn't really indicate success or failure.

Not outputting it in the first place is another valid solution.